apis/python/src/tiledbsoma/reindexer.cc

```diff
@@ -148,15 +148,17 @@ void load_reindexer(py::module& m) {
         // looked up values into previously allocated array (works for the
         // cases in which python and R pre-allocate the array)
         .def(
-            "get_indexer",
+            "get_indexer_general",
```

Comment:
Are we able to just do `.def("get_indexer_general", &IntIndexer::get_indexer_general)` now?